src/Morse/MorseStream.tsx

```diff
@@ -151,18 +151,27 @@ class MorseStream extends LocalStorageComponent<Props, State, SavedState> {
       return;
     }
 
-    let handled = false;
-
-    if (ev.keyCode === 13) { // Enter
-      this.onNextClick();
-      handled = true;
-    } else if (ev.charCode === 45 || ev.charCode === 106) { // '-' or 'J'
-      this.onDashClick();
-      handled = true;
-    } else if (ev.charCode === 46 || ev.charCode === 107) { // '.' or 'K'
-      this.onDotClick();
-      handled = true;
-    }
+    let handled = true;
+    switch (ev.keyCode) {
+      case 45: // '-'
```

Comment:
Is there some reason why we're using keyCode?  Looks like keyCode is deprecated, and there's key.data has the actual key pressed (eg '-') in it. 

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key

Reply:
Because web development is messy, and there are usually two choices: The deprecated one that works everywhere, and the new one that doesn't work everywhere.

https://medium.com/@uistephen/keyboardevent-key-for-cross-browser-key-press-check-61dbad0a067a

I'll see if I can tweak this a bit.....

Reply:
Added support for event.key, tested in Firefox and Chromium to ensure everything was still working.